src/devtools/client/debugger/src/selectors/isSelectedFrameVisible.ts

```diff
@@ -6,15 +6,15 @@ import type { UIState } from "ui/state";
 
 import { getSelectedLocation } from "ui/reducers/sources";
 
-import { getSelectedFrame } from "./pause";
+import { getSelectedFrameAsync } from "../selectors/pause";
 
 /*
  * Checks to if the selected frame's source is currently
  * selected.
  */
-export function isSelectedFrameVisible(state: UIState) {
+export async function isSelectedFrameVisible(state: UIState) {
   const selectedLocation = getSelectedLocation(state);
-  const selectedFrame = getSelectedFrame(state);
+  const selectedFrame = await getSelectedFrameAsync(state);
```

Comment:
Are async "selectors" common?

Reply:
Not really. Would you prefer to remove that "selector" and do the `getSelectedFrameId()` / `getPauseFrameAsync()` combo at each call site?

Reply:
I think a helper function is fine. I just think it's confusing when we have things called "selectors" that do more than just synchronously extract values from Redux state– but @markerikson probably has a more informed opinion here.

Reply:
Yeah, we've recommended `selectX` as a preferred naming convention for selectors for the last few years, but this codebase was already using `getX` instead.

This _is_ becoming confusing now that we're also using `getSomeThingAsync` as a naming convention for our cache APIs...